tests/gen_cert_test.py

```diff
@@ -107,6 +105,7 @@ def test_cert_names():
     # XXX: This is meant to catch unicode rendering problems, but does it?
     course_id = settings.CERT_DATA.keys()[0]
     for name in NAMES:
+        print 'asdf'
```

Comment:
Please clearly state the issue and only the issue.   This one may be obvious but many times it's not clear what 'this' means.